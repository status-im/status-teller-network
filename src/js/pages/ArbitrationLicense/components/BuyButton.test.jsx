import React from "react";
import { shallow } from "enzyme";

import LicenseBuy from "./BuyButton";

describe('LicenseBuy', () => {
  it('should render correctly when disabled', () => {
    const component = shallow(<LicenseBuy disabled={false} onClick={() => undefined}/>);
  
    expect(component).toMatchSnapshot();
  });

  it('should render correctly when enabled', () => {
    const component = shallow(<LicenseBuy disabled={true} onClick={() => undefined}/>);
  
    expect(component).toMatchSnapshot();
  });
});
