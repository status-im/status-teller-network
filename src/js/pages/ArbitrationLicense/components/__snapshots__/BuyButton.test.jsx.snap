// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LicenseBuy should render correctly when disabled 1`] = `
<Row
  className="mt-2"
  tag="div"
>
  <Col
    className="text-center"
    tag="div"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
      ]
    }
    xs={12}
  >
    <Button
      color="primary"
      disabled={false}
      onClick={[Function]}
      tag="button"
    />
  </Col>
</Row>
`;

exports[`LicenseBuy should render correctly when enabled 1`] = `
<Row
  className="mt-2"
  tag="div"
>
  <Col
    className="text-center"
    tag="div"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
      ]
    }
    xs={12}
  >
    <Button
      color="primary"
      disabled={true}
      onClick={[Function]}
      tag="button"
    />
  </Col>
</Row>
`;
